config/puma.rb

```diff
@@ -1,6 +1,6 @@
 require './config/config'
 environment Config.rack_env
-port Config.port
+port Config.http_port
```

Comment:
`$PORT` is a special env variable handled by Foreman, so this is a breaking change.